ui/src/components/ProvisionD-Config.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
I don't know about mixing PascalCase and kebab-case here. Maybe `ProvisionDConfig.vue` will be more consistent with the other filenames.

Reply:
Name changed to ProvisionDConfig.vue